include/class.task.php

```diff
@@ -1062,6 +1062,14 @@ function getVar($tag) {
             break;
         case 'last_update':
             return new FormattedDate($this->last_update);
+        case 'description':
+            $entry = $this->getThread()->getEntries()->filter(array(
```

Comment:
This duplicates what's already in `class.thread.php` 

```php
return Format::display($this->getThread()->getVar('original') ?: '');
```

Reply:
can't believe I missed that, thank you